e2e/pnpm_lockfiles/base/package.json

```diff
@@ -20,6 +20,9 @@
     "pnpm": {
         "patchedDependencies": {
             "meaning-of-life@1.0.0": "patches/meaning-of-life@1.0.0-pnpm.patch"
-        }
+        },
+        "onlyBuildDependencies": [
+            "@aspect-test/c"
```

Comment:
This is required for v9, and we have a test asserting the lifecycle targets exist.

We currently have no tests asserting that other packages *do not* have lifecycle targets though, that would be more then the simple `build_test`...